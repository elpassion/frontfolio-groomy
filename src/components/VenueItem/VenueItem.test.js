import React from 'react';
import { shallow } from 'enzyme';

import { VenueItem } from './index';

const mockDetails = {
  id: 1,
  name: 'The Bark Club',
  address: '8 Brondesbury Park',
  distance: '1.2km',
  price: '$120',
};

it('renders correctly', () => {
  const wrapper = shallow(<VenueItem details={mockDetails} />);
  expect(wrapper).toMatchSnapshot();
});
