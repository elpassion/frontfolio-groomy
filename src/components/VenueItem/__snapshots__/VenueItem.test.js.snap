// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<VenueItemLink
  onClick={[Function]}
>
  <PhotoWrapper>
    <VenuePhoto
      src="/images/venues/salon-1.jpg"
      style={
        Object {
          "borderRadius": null,
          "height": null,
          "left": null,
          "opacity": null,
          "top": null,
          "width": null,
        }
      }
    />
  </PhotoWrapper>
  <VenueMeta>
    <VenueName>
      The Bark Club
    </VenueName>
    <VenueAddress>
      8 Brondesbury Park
    </VenueAddress>
    <VenueDetailsWrapper>
      <MetaItem>
        <StyledRatingIcon />
         
        5.0
      </MetaItem>
      <MetaItem>
        <StyledLocationIcon />
         
        1.2km
      </MetaItem>
      <MetaItem>
        <StyledPriceIcon />
         
        $120
      </MetaItem>
    </VenueDetailsWrapper>
  </VenueMeta>
</VenueItemLink>
`;
